packages/core/src/utils/utils.ts

```diff
@@ -121,6 +119,9 @@ export const generateRandomId = (letterCount: number): string => {
   const NUMS = '0123456789';
   const ALNUM = NUMS + LETTERS;
 
+  const times = (n: number, callback: () => string) => Array.from({ length: n }, callback);
+  const random = (min: number, max: number) => Math.floor(Math.random() * (max - min + 1)) + min;
```

Comment:
Why do we calculate it like this?

Reply:
Not sure Im following, are you asking about the internal implementation of random? Its fairly typical for a js random function that i've seen, but if you have suggestions im up for it